lib/std/builtin.zig

```diff
@@ -1101,5 +1103,240 @@ pub noinline fn returnError() void {
     st.index += 1;
 }
 
-const std = @import("std.zig");
-const root = @import("root");
+pub const Clobbers = switch (builtin.cpu.arch) {
+    .x86_64 => packed struct {
```

Comment:
This is missing:

* 16 extra APX registers
* 8 AVX-512 mask registers
* 8 AMX tile registers and 1 config register
* SSP register